### PR TITLE
Tooltip to inform of reason for disabled action

### PR DESCRIPTION
**Pull request by CakeCrusher**
## Note
I added the following TODOs:
- `{/* TODO: incorporate localize */}` I did not incorporate the translations for this tooltip message
- `{/* TODO: refactor to use https://atomiks.github.io/tippyjs/ */}` Using this custom tooltip UI is convoluted tippyjs could significantly simplify this.
## Summary
when creating an action for an assistant the button is sometimes disabled if the assistant has not yet been created therefore without an explanation of why it's disabled this PR informs the user of why the button is disabled.

## Change Type

Please delete any irrelevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Translation update

## Testing
you click on assistance and then you go to assistant builder where you will see that for an assistant that is already created the action button is activated but if you try and create a new action the action button defaults to disabled that is when the tooltip shows why it is disabled.

## Checklist

Please delete any irrelevant options.

- [x] My code adheres to this project's style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented in any complex areas of my code
- [x] My changes do not introduce new warnings
- [x] Local unit tests pass with my changes

### CODE REVIEW

**Comment by CakeCrusher** [comment]
@danny-avila

File: client/src/components/SidePanel/Builder/AssistantPanel.tsx [hunk 1]
```diff
@@ -385,24 +387,44 @@ export default function AssistantPanel({
...
-                  className="btn border-token-border-light relative mt-2 h-8 rounded-lg bg-transparent font-medium hover:bg-gray-100 dark:hover:bg-gray-800"
-                >
-                  <div className="flex w-full items-center justify-center gap-2">
-                    {localize('com_assistants_add_actions')}
-                  </div>
-                </button>
+                <div className="relative mt-2 h-8 rounded-lg bg-transparent font-medium">
+                  <TooltipProvider delayDuration={0}>
+                    {' '}
+                    {/* TODO: refactor to use https://atomiks.github.io/tippyjs/ */}
```
Comment (danny-avila) [inline client/src/components/SidePanel/Builder/AssistantPanel.tsx]: ?

File: client/src/components/SidePanel/Builder/AssistantPanel.tsx [hunk 2]
```diff
@@ -385,24 +387,44 @@ export default function AssistantPanel({
...
+                          }}
+                          className="btn border-token-border-light relative rounded-lg bg-transparent font-medium hover:bg-gray-100 dark:hover:bg-gray-800"
+                        >
+                          <div className="flex w-full items-center justify-center gap-2">
+                            {localize('com_assistants_add_actions')}
+                          </div>
+                        </button>
+                        {disabledActionTip && (
+                          <div className="absolute bottom-10 left-0 z-10 rounded bg-white p-2 shadow dark:bg-gray-800">
+                            {/* TODO: incorporate localize */}
```
Comment (danny-avila) [inline client/src/components/SidePanel/Builder/AssistantPanel.tsx]: yes, please incorporate it

**Review by danny-avila — CHANGES_REQUESTED** [review]

File: client/src/components/SidePanel/Builder/AssistantPanel.tsx (hunk 1 above)
Comment (danny-avila) [reply client/src/components/SidePanel/Builder/AssistantPanel.tsx]: No thank you on `tippyjs`

**Comment by CakeCrusher** [comment]
Moved to #4306

**Pull request closed without merging**